public/app/features/transformers/docs/content.ts

```diff
@@ -581,7 +606,7 @@ export const transformationDocsContent: TransformationDocsContentType = {
     name: 'Grouping to matrix',
     getHelperDocs: function () {
       return `
-  Use this transformation to combine three fields-that will be used as input for the **Column**, **Row**, and **Cell value** fields-from the query output, and generate a matrix. This matrix will be calculated as follows:
+  Use this transformation to combine three fields, which will be used as input for the **Column**, **Row**, and **Cell value** fields from the query output, and generate a matrix. The matrix is calculated as follows:
```

Comment:
I think em-dashes here improve the readability of this sentence. They were placed incorrectly before, but this placement should make this sentence a bit easier to understand.
```suggestion
  Use this transformation to combine three fields—which are used as input for the **Column**, **Row**, and **Cell value** fields from the query output—and generate a matrix. The matrix is calculated as follows:
```